test/e2e/e2e_init.go

```diff
@@ -0,0 +1,28 @@
+// +build e2e
+
+package e2e
+
+import (
+	"os"
+	"os/exec"
+	"strings"
+)
+
+var fluxBinary string
+
+func init() {
```

Comment:
Instead of using `init` the usage of [`TestMain`](https://pkg.go.dev/testing#hdr-Main) might be preferable here as it is more explicit.